### PR TITLE
Correct Missing Type Annotations

### PR DESCRIPTION
**Pull request by tcfranks**

### CODE REVIEW

**Comment by tcfranks** [comment]
so question on this error:  black reformats (locally and remotely line 177, which is 

def _unpack_and_scale(self, top: int, bottom: int) -> float:  # pylint: disable=no-self-use

into 
def _unpack_and_scale(
        self, top: int, bottom: int
    ) -> float:  # pylint: disable=no-self-use

and this reformat throws the error - so is there a way around this isssue?

**Comment by tcfranks** [comment]
well dang, I thought I had it....

**Comment by tekktrik** [comment]
Hey @tcfranks I responded to you in the Adafruit Discord about this, but let me know if you need me to repost my response here!

**Comment by tcfranks** [comment]
ok, last attempt was to add to pylintrc "load-plugins=pylint.extensions.no_self_use"

and bracket the function with disable / enable statements.  pylint still not happy about that.....

File: adafruit_tlv493d.py [hunk 1]
```diff
@@ -175,10 +175,9 @@ def magnetic(self) -> Tuple[float, float, float]:
         )
 
     # pylint: disable=no-self-use
-    def _unpack_and_scale(
-        self, top: int, bottom: int
-    ) -> float:
+    def _unpack_and_scale(self, top: int, bottom: int) -> float:
```
Comment (tekktrik) [inline adafruit_tlv493d.py]: Try:

```python
    def _unpack_and_scale(  # pylint: disable=no-self-use
        self, top: int, bottom: int
    ) -> float:
```

**Review by tekktrik — COMMENTED** [review]
Haven't had a chance to look, but this will likely solve the issue:

File: adafruit_tlv493d.py (hunk 1 above)
Comment (tcfranks) [reply adafruit_tlv493d.py]: well, I swear I tried that before.  whatever changed - had put pylintrc back to it's original state.  Looks like it passed this time!

**Comment by tcfranks** [comment]
oops , probably should have done a new branch?

**Comment by FoamyGuy** [comment]
> oops , probably should have done a new branch?

if you mean a branch other than `tcfranks:main` it is probably a bit easier to keep things straight if you make a new branch in your fork instead of using main. So something like `tcfranks:typing_info` would be a good branch name for this PR. For me this mostly tends to come up if you've already submitted one PR for a library where you worked on the main branch, and then you want to start or submit a different PR that deals with something else. It will be slightly harder to start that 2nd new branch in the right spot since you've used `main` for the first one. Not impossible, just a bit different. 

However the names themselves aren't actually "special" as far as I understand it. There is a default branch setting at the repo level, and it happens to be set to "main" on this repo, but it could be anything else as well and things would still work. By itself, the name "main" doesn't mean anything extra compared to other names. 

It's a good habit to get into (choosing descriptive names for your branches, and making a new branch for each PR.). But it's not strictly necessary, and if you sometimes forget it's okay.

File: adafruit_tlv493d.py [hunk 2]
```diff
@@ -119,36 +127,36 @@ def __init__(self, i2c_bus, address=_TLV493D_DEFAULT_ADDRESS, addr_reg=0):
...
 
-    def _get_read_key(self, key):
+    def _get_read_key(self, key: int) -> int:
         read_byte_num, read_mask, read_shift = self.read_masks[key]
         raw_read_value = self.read_buffer[read_byte_num]
         write_value = (raw_read_value & read_mask) >> read_shift
         return write_value
 
-    def _set_write_key(self, key, value):
+    def _set_write_key(self, key: int, value: int) -> None:
```
Comment (tekktrik) [inline adafruit_tlv493d.py]: `key` should be `str` based on the `dict` it's accessing.

File: adafruit_tlv493d.py [hunk 3]
```diff
@@ -119,36 +127,36 @@ def __init__(self, i2c_bus, address=_TLV493D_DEFAULT_ADDRESS, addr_reg=0):
...
 
-    def _setup_write_buffer(self):
+    def _setup_write_buffer(self) -> None:
         self._read_i2c()
         for key in ["RES1", "RES2", "RES3"]:
             write_value = self._get_read_key(key)
             self._set_write_key(key, write_value)
 
-    def _get_read_key(self, key):
+    def _get_read_key(self, key: int) -> int:
```
Comment (tekktrik) [inline adafruit_tlv493d.py]: `key` should be `str` since `self.read_masks` is a `dict` of with `str` keys.

**Review by tekktrik — CHANGES_REQUESTED** [review]
Couple bits of feedback!

**Comment by tcfranks** [comment]
ready for re-review after fixing issues pointed out by tekktrik

**Review by tekktrik — APPROVED** [review]
Sweeto, thanks!

**Pull request merged**